### DISCUSSION TITLE
Quarto documentation

### DISCUSSION THREAD

**Discussion by kostadinoff** [quarto-dev/quarto-cli#5412] [Q&A] [answered]
I find the quarto website amazing. However i would like to have the quarto documentation as a pdf file. Is there anything like that?

**Comment by mcanouil** [comment]
Not at this moment unfortunately.
This is because Quarto documentation is a website, and website are HTML.

**Reply by mcanouil** [reply, marked as answer]
Using the GitHub repository, you can preview (or render) the documentation locally, if the need for PDF was to have the documentation locally.
